### PR TITLE
[#5745] validate account recover result

### LINKED ISSUE #5745: Recovering the same account with a different password is stuck
### Description

*Type*: Bug

*Summary*: When I recover an account that already exists in the app but I use a different password, I am stuck on the recover screen.

#### Expected behavior
I should get a notification that such an account already exists and the password does not match.

If I forgot the password, what should I do? Can I delete the previous account and recover it with a different password?

#### Actual behavior
I am stuck on the recover screen without any message and I need to kill the app (remove it from the background) to try to recover a different account or recover it with a correct password.

### Reproduction

- Open Status,
- If you don't have any accounts, create one,
- Logout from the current account and go to the recover screen,
- Recover the existing account but use a different password this time.

### Additional Information
* Status version: 0.9.27, 0.9.28 nightly
* Operating System: Android | iOS

### PR DESCRIPTION
**Pull request by dmitryn**
fixes #5745

### Summary:

Validate account recover result and show error to the user in case of unsuccessful recovery (password is not valid).

status: ready

### CODE REVIEW

**Review by yenda — APPROVED** [review]

**Review by rasom — APPROVED** [review]

File: src/status_im/translations/en.cljs [hunk 1]
```diff
@@ -401,6 +401,7 @@
    :twelve-words-in-correct-order        "12 words in correct order"
    :enter-12-words                       "Enter the 12 words of your recovery phrase, separated by single spaces"
    :recover-password-too-short           "Password is too short"
+   :recover-password-invalid             "Password is not valid"
```
Comment (rasom) [inline src/status_im/translations/en.cljs]: please change it in `translations/en.json`

**Review by mandrigin — APPROVED** [review]

File: src/status_im/translations/en.cljs (hunk 1 above)
Comment (dmitryn) [reply src/status_im/translations/en.cljs]: @rasom thanks, migrated to json.

**Comment by lukaszfryc** [comment]
@dmitryn 2 things I'd change:

1. Make sure the tooltip is not overlapped, see:



2. Change the copy to "This account already exists but passwords do not match"

**Review by flexsurfer — APPROVED** [review]

**Comment by dmitryn** [comment]
@lukaszfryc 
1. Was fixed by https://github.com/status-im/status-react/commit/8fa0522bada0e2923bc5883553423e206a30034b (merged it here and to develop)
2. Changed.

Please take a look.

**Comment by lukaszfryc** [comment]
@dmitryn please update the changelog before merging.

**Pull request merged**